bin/yk-config

```diff
@@ -143,7 +143,11 @@ handle_arg() {
             OUTPUT="${OUTPUT} -Wl,--mllvm=--yk-shadow-stack"
             # Encode additional locations in stackmaps.
             OUTPUT="${OUTPUT} -Wl,--mllvm=--yk-stackmap-add-locs"
-
+            # Use software-tracer pass
+            # Note: We can't run yk-basicblock-tracer pass with HardwareTracer
```

Comment:
I think the "note line" is irrelevant: it's kind of obvious that the "swt" pass only makes sense with "swt".

Reply:
removed 👉 de4a1896b9c2d48a7f9aa1bba688520a82bcc562